atom/browser/web_contents_preferences.cc

```diff
@@ -157,6 +158,17 @@ void WebContentsPreferences::AppendExtraCommandLineSwitches(
 
   // The initial visibility state.
   NativeWindow* window = NativeWindow::FromWebContents(web_contents);
+
+  // Use embedder window for webviews
+  if (guest_instance_id && !window) {
```

Comment:
The `guest_instance_id` needs to be initialized above, otherwise it could be filled with random data.